src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/httpRenderer.jsx

```diff
@@ -10,11 +12,17 @@ class HttpRenderer extends React.Component {
   };
 
   renderUrl = url => {
-    return typeof url === 'string' && url.match(/^https?:\/\//) ? (
-      <a href={url}>{url}</a>
-    ) : (
-      <em>{url}</em>
-    );
+    if (typeof url === 'string') {
+      return url.match(/^https?:\/\//) ? <a href={url}>{url}</a> : <em>{url}</em>;
+    }
+
+    Sentry.withScope(scope => {
+      scope.setExtra('url', url);
+      scope.setLevel('info');
+      Sentry.captureException(new Error('Invalid breadcrumb URL'));
+    });
+
+    return t('Invalid URL');
```

Comment:
Would it make sense to stringify the value and still render it?

Reply:
What does a URL that's not a string look like?

Reply:
There was one case where an empty object was sent, not sure what else comes up.